external/fv3fit/fv3fit/_shared/models.py

```diff
@@ -177,3 +179,52 @@ def load(cls, path: str) -> "EnsembleModel":
         models = [io.load(path) for path in config["models"]]
         reduction = config["reduction"]
         return cls(models, reduction)
+
+
+@io.register("output_transformed_model")
+class TransformedPredictor(Predictor):
```

Comment:
This is very similar to the `DerivedModel`, the important difference being that transformation names are used for configuration, instead of variable names. We might want to switch over to using this instead of `DerivedModel`, but the `TransformedPredictor` is not currently as fully featured, since I have not implemented the concept of "required variables" for the data transforms, and so we cannot do things like adding variable names to the model inputs.